rdr_service/genomic/genomic_data.py

```diff
@@ -21,6 +21,15 @@ def __init__(self, input_manifest=None):
             'gsm': aliased(GenomicSetMember),
         }
 
+        self.subqueries = {
+            'aw3_wgs_parent_sample_id': (
+                sqlalchemy.select([GenomicSetMember.gcManifestParentSampleId]).select_from(GenomicSetMember).where(
+                    (GenomicSetMember.genomeType == "aou_array") &
+                    (GenomicSetMember.aw3ManifestJobRunID.isnot(None))
+                )
+            )
+        }
+
```

Comment:
:nit would you mind running the code format tool on this or breaking the lines for it.

Reply:
Reformatted.